__tests__/app.test.js

```diff
@@ -62,4 +55,27 @@ describe("GET article by id", ()=>{
             expect(article).toHaveProperty("article_img_url")
         })
     })
-})
\ No newline at end of file
+});
+
+describe("GET articles", ()=>{
+    test("",()=>{
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((responce) => {
+            expect(Array.isArray(responce.body.articles)).toBe(true);
```

Comment:
There is an assertion that needs to be here 👀 If the response body.articles is an empty array, this assertions will pass but the assertions in the .forEach won't run, and the test will automatically pass 🫨 So we need an assertion to check that the array has a `length` property.